extension/intents/music/music.js

```diff
@@ -71,6 +71,9 @@ async function pauseAnyButTab(context, tabId) {
 intentRunner.registerIntent({
   name: "music.play",
   async run(context) {
+    if (context.parameters.prefixQuery !== undefined) {
+      context.slots.query += " " + context.parameters.prefixQuery;
```

Comment:
This will probably come up with the correct search result, but it will turn a phrase like `play trailer of avatar` into a search query `avatar trailer`. It would be better if this was `context.slots.query = context.parameters.prefixQuery + " " + context.slots.query`

Reply:
@ianb I have made the changes as asked by you and also tested it and its working as per expectation.
Thanks.